src/viewport.h

```diff
@@ -6,6 +6,21 @@
 
 struct imv_viewport;
 
+enum scaling_mode {
+  SCALING_NONE,
+  SCALING_DOWN,
+  SCALING_FULL,
+  SCALING_CROP,
+  SCALING_MODE_COUNT
+};
+
+static const char *scaling_label[] = {
```

Comment:
I think you want to declare this in the header, but give its value in the C file, or you get compile warnings like:

```
In file included from src/viewport.c:1:
src/viewport.h:17:20: warning: ‘scaling_label’ defined but not used [-Wunused-variable]
   17 | static const char *scaling_label[] = {
      |                    ^~~~~~~~~~~~~
```